aws_xray_sdk/core/recorder.py

```diff
@@ -284,6 +284,8 @@ def begin_subsegment(self, name, namespace='local'):
 
         :param str name: the name of the subsegment.
         :param str namespace: currently can only be 'local', 'remote', 'aws'.
+        :param bool sampling: sampling decision for the subsegment being created,
+            defaults to True
```

Comment:
This `sampling` parameter is not the same in meaning with the `sampled` flag on an entity.
`sampling=True` here means that the subsegment will inherit the sampling decision of its parent entity, and could either be sampled or not. 
`sampling=False` would mean that the subsegment will not inherit the sampling decision and will be unsampled.

I hope I was able to explain the difference. Please reword the description. :)